packages/p2p/src/stores/my-profile-store.js

```diff
@@ -468,11 +468,11 @@ export default class MyProfileStore extends BaseStore {
 
     getSettings() {
         requestWS({ get_settings: 1 }).then(response => {
-            const { get_settings } = response;
-            if (!response.error) {
-                this.setFullName(`${get_settings.first_name} ${get_settings.last_name}`);
+            const { get_settings } = response || {};
```

Comment:
Do we need the `{}` default here, if `response` is undefined and defaults to `{}` then the first if statement could still be executed

Reply:
yeahhhh this is an issue i saw in other places, we can try to this as well 👍